tests/Ua/StudyIntegrationTest.php

```diff
@@ -200,6 +167,7 @@ public function testGetAllStudyWithLeaderShouldReturn200()
         $body = json_decode($response->getBody());
 
         $this->assertEquals(3, count($body->content));
+        $this->assertNotNull($body->content[1]->documents);
```

Comment:
Une boucle for pour tester que tous les documents ne sont pas null serait mieux non ?

Reply:
Yes